### ISSUE TITLE
大脑爆炸了can i use it to auto annotation the image for create a dataset to train the custom Yolo v8 model? 

### ISSUE THREAD

**Issue by Mayuyu007123** [AILab-CVC/YOLO-World#191]
the labels be like this
![螢幕擷取畫面 2024-03-28 032350](https://github.com/AILab-CVC/YOLO-World/assets/102665743/88607df1-c1f1-440f-a74c-b6bcbf5f378a)

**Comment by wondervictor** [comment]
Hi @Mayuyu007123, firstly, could you provide more details about your `demo`? 
Secondly, could you share your sample image?
Thirdly, YOLO-World exactly can be used as an automatic annotator and an open-source project has involved it, see https://github.com/AILab-CVC/YOLO-World/issues/120. However, in my opinion, fine-tuning YOLO-World on your data with fewer epochs can obtain better performance on YOLOv8, you can refer to [configs/finetune_coco](https://github.com/AILab-CVC/YOLO-World/tree/master/configs/finetune_coco)

**Comment by ljyan93** [comment]
hey @wondervictor could you point us which model & config & script (with NMS) to reproduce the huggingface demo locally?

thanks in advance.
